libqtile/core/manager.py

```diff
@@ -917,15 +918,15 @@ def cmd_groups(self) -> dict[str, dict[str, Any]]:
         """
         return {i.name: i.info() for i in self.groups}
 
-    def cmd_display_kb(self, *args) -> str:
+    def cmd_display_kb(self) -> str:
```

Comment:
I'm not clear how `args` may have been used in this method but, if it's no longer required, should we still accept it and just provide a deprecation warning rather than having a breaking change?

Reply:
I don't see why anybody would have ever used it as it never did anything. TBH I'd like it if CI caught this when it was added, because it's an unused variable. `vulture` would pick it up but we disabled it in CI because it is very verbose, I think `pylint` was possibly also disabled for the same reason, so that's probs why it passed. I was playing around with vulture again recently (trying to find dead code) and it I stand by keeping it disabled in CI, but perhaps we should reassess using pylint in CI - maybe we can configure it to pick up just the little things we might want to avoid (like these unused arguments). Anyway, that's unrelated to this PR, but I think it'd be fine to not bother with deprecating this.

Reply:
I'm fine for this to be merged but it might be worth rerunning CI as I think something's gone wrong somewhere so should check this is still ok